src/domains/citizen-auth-app/07_function_fast_login.tf

```
@@ -0,0 +1,258 @@
+locals {
+  function_fast_login = {
+    app_settings = {
+      FUNCTIONS_WORKER_PROCESS_COUNT = 4
+      NODE_ENV                       = "production"
+
+      // Keepalive fields are all optionals
+      FETCH_KEEPALIVE_ENABLED             = "true"
+      FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL   = "110000"
+      FETCH_KEEPALIVE_MAX_SOCKETS         = "40"
+      FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
+      FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
+      FETCH_KEEPALIVE_TIMEOUT             = "60000"
+
+      COSMOS_DB_NAME           = "citizen-auth"
+      COSMOS_DB_URI            = data.azurerm_cosmosdb_account.cosmos_citizen_auth.endpoint
+      COSMOS_DB_KEY            = data.azurerm_cosmosdb_account.cosmos_citizen_auth.primary_key
+      COSMOS_CONNECTION_STRING = format("AccountEndpoint=%s;AccountKey=%s;", data.azurerm_cosmosdb_account.cosmos_citizen_auth.endpoint, data.azurerm_cosmosdb_account.cosmos_citizen_auth.primary_key)
+    }
```

regarding the above comment
```suggestion
      // --------------------------
      //  LolliPoP Consumer config
      // --------------------------
      LC_ASSERTION_CLIENT_BASE_URL         = "https://api.io.pagopa.it"
      LC_ASSERTION_CLIENT_SUBSCRIPTION_KEY = data.azurerm_key_vault_secret.fast_login_subscription_key.value
    }
```

---

Fixed in 263c60e